scripts/devbox-init-hook.sh

```diff
@@ -2,29 +2,16 @@
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 SRC_DIR="$(dirname $DIR)"
 
-# Set install path
-INSTALL_DIR="$HOME/.local/bin"
-mkdir -p "$INSTALL_DIR"
-
-# Check if envsubst.old exists
-if [[ ! -f "/usr/bin/envsubst.old" ]]; then
-    echo "envsubst.old does not exist. Proceeding with installation..."
+# Set install path to devbox packages directory
+INSTALL_DIR="$DEVBOX_PACKAGES_DIR/bin"
```

Comment:
I would probably install envsubst and clusterawsadm into the `bin` folder of the project (like we do the tools installed from the makefile) instead of the devbox packages directory. Feels like that devbox package directory should be left to devbox.